src/routes/Dashboard/Page.svelte

```diff
@@ -1,12 +1,75 @@
-<script>
+<script lang="ts">
+    import COLORS from 'tailwindcss/colors';
+    import { format } from 'date-fns';
+    import MdDashboard from 'svelte-icons/md/MdDashboard.svelte';
+    import MdSettings from 'svelte-icons/md/MdSettings.svelte';
+
     import Layout from '../../components/Layout.svelte';
+    import Select from '../../components/Select/Select.svelte';
+    import Spacer from '../../components/Spacer.svelte';
+    import TabGroup from '../../components/Tab/TabGroup.svelte';
+    import TabItem from '../../components/Tab/TabItem.svelte';
+    import Text from '../../components/Text.svelte';
     import Background from './Background.svelte';
+    import Valve from '../../components/Valve.svelte';
+    import type { Option } from '../../components/Select/types.ts';
+
+    import {
+        LOGS,
+        LOG_TYPE_TEXT,
+        LOG_TYPE_COLORS,
+        OPTIONS,
+    } from './constants.ts';
+
+    let value: Option = OPTIONS[0] || '';
+
+    let active = true;
 </script>
 
 <Layout>
-    <h1>Dashboard Page</h1>
-    <Background />
+    <div class="wrapper flex flex-col items-center gap-4 p-4 text-xs">
+        <span class="w-full">
+            <h2>Welcome</h2>
+            <h1>Some-Dood</h1>
+        </span>
+        <Spacer --length="25cqh" />
+        <span class="flex w-full justify-between">
+            <Select name="mac" options={OPTIONS} bind:value disabled>
+                Device Mac:
+            </Select>
```

Comment:
If we insist on having the ability to automatically select the first `<option>`, consider using the `selected` attribute instead and react to the defaulted selection via `bind:value`. Should there be 0 entries in the `<select>` group, there will be no `selected` `<option>`, which resolves `value` as `null` (i.e., must manually render a special something instead).